warehouse/ingest-scripts/src/main/resources/bin/system/start-ingest.sh

```diff
@@ -29,7 +31,12 @@ else
   trap 'rm -f "$ingestHost"; exit $?' INT TERM EXIT
   echo $INGEST_HOST > $ingestHost
 
-  pdsh -f 25 -w ^${ingestHost} "$INGEST_BIN/ingest/start-ingesters.sh $FORCE" < /dev/null
+  localhost=$(hostname -s)
+  rm -rf /tmp/pdsh_log/${script_name}/$$
+  pdsh -f 25 -w ^${ingestHost} "$INGEST_BIN/ingest/start-ingesters.sh $FORCE" < /dev/null \
```

Comment:
It seems this pattern is repeated a lot. Maybe it's worth pulling it into a bash function and call it from each script?

Reply:
Good idea, done in next commit.